mk/containerized.mk

```
@@ -19,9 +19,13 @@ pull: ## Download SVC images
 build:  ## Build SVC images
 build: DOCKER_COMPOSE_ENV = DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
 
-push: ## Push SVC container images to a registry. Requires previous "docker login"
+GIT_LATEST_TAG != git describe --tags --abbrev=0
```

I'd either rename this as:
```suggestion
NVMEOF_VERSION != git describe --tags --abbrev=0
```
However, we tried to remove all nvmeof-specific references from this file (it behaves as a abstract makefile for docker-compose commands, without any knowledge of the nvmeof specifics).

So, I'd rather define this in the `Makefile`:
```make
build up demo run logs push: export NVMEOF_VERSION != git describe --tags --abbrev=0
```

---

done